### ISSUE TITLE
AVT 3 - Tile Selectable: VoiceOver does not announce when a selectable tile is checked or unchecked

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2241]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
With VoiceOver enabled tab to the selectable tile that is checked (selectable tile 2)

## Expected Result
VoiceOver announces when the selectable tile is checked and unchecked. 

Note: Based on the selectable tiles type="radio" the selectable component should follow the [ARIA Authoring Practices 1.1 design pattern for radio group](https://www.w3.org/TR/wai-aria-practices-1.1/examples/radio/radio-1/radio-1.html) which requires  aria-checked attribute set on the selectable tiles.

**Comment by dakahn** [comment]
Testing on latest macOS and chrome here's what I have:

VO on selected tile:
<img width="659" alt="Screen Shot 2019-04-22 at 12 22 39 AM" src="https://user-images.githubusercontent.com/40970507/56484386-f3384780-6494-11e9-86ce-e5527c1f5602.png">

VO when another tile is selected:
<img width="652" alt="Screen Shot 2019-04-22 at 12 23 28 AM" src="https://user-images.githubusercontent.com/40970507/56484393-fe8b7300-6494-11e9-962d-5757c3ddd84b.png">

Maybe specifically we need "checked" and "unchecked" here _according to the letter of the law_, but since the component is described by VO as a "selectable tile" describing it as "selected" seems to make more sense.

**Comment by snidersd** [comment]
@dakahn The issue is not that about the word "selected" vs "checked". When I test with VO the announcement of the selected state is inconsistent. For example, If I tab to the default tile (2) VO announces that it is selected... as in your screenshot. If I then tab to the next tile and hit the Enter key to select it, VO announces "You are currently on a group. To interact with items in this group press Control-Option-Shift-Down Arrow."
<img width="920" alt="Screen Shot 2019-04-22 at 8 07 11 AM" src="https://user-images.githubusercontent.com/21676914/56502338-990ca600-64d7-11e9-87d0-8aac769e40fd.png">

 In order to hear VO announce the selected state, I have to tab back to the selected radio button. In the ARIA Authoring practices example, the state is announced as soon as the selection is made.
